spacy_cld/spacy_cld.py

```diff
@@ -17,7 +17,10 @@ def get_scores(text, cld_results=None):
 
 
 def detect_languages(text):
-    _, _, results = detect(text.text)
+    try:
+        _, _, results = detect(text.text)
+    except:
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) recommends against catching _all_ exceptions in `except` blocks - this can make it a bit too easy to squash errors that a user would want to see.

Instead, if you import the error being thrown with `from pycld2 import error as pycld_error`, then you can change this line to `except pycld_error as err:`, which is quite a bit more specific.